src/router/index.js

```diff
@@ -1,33 +1,26 @@
 import Vue from 'vue'
-import Router from 'vue-router'
-import store from '@/store'
-import Dashboard from '@/components/Dashboard.vue'
-import Login from '@/components/Login.vue'
+import VueRouter from 'vue-router'
 
-Vue.use(Router)
+import routes from './routes'
 
-export default new Router({
-  mode: 'history',
-  base: __dirname,
-  routes: [
-    { path: '/dashboard', component: Dashboard, beforeEnter: requireAuth },
-    { path: '/login', component: Login },
-    { path: '/logout',
-      beforeEnter (to, from, next) {
-        store.dispatch('logout')
-        next('/')
-      }
-    }
-  ]
-})
+Vue.use(VueRouter)
```

Comment:
I appreciate this approach here, of moving routing configuration in another file, kudos!

Reply:
Thanks :smile: